### PR TITLE
Math notation 2

### PR DESCRIPTION
**Pull request by hollasch**
Update math notation.

- Eliminate vector barbs
- Points get uppercase bold
- Vectors get lowercase bold

### CODE REVIEW

File: books/RayTracingInOneWeekend.html [hunk 1]
```diff
@@ -386,19 +386,20 @@
...
 The one thing that all ray tracers have is a ray class, and a computation of what color is seen
-along a ray. Let’s think of a ray as a function $\mathbf{p}(t) = \mathbf{a} + t \vec{\mathbf{b}}$.
-Here $\mathbf{p}$ is a 3D position along a line in 3D. $\mathbf{a}$ is the ray origin and
-$\vec{\mathbf{b}}$ is the ray direction. The ray parameter $t$ is a real number (`double` in the
-code). Plug in a different $t$ and $p(t)$ moves the point along the ray. Add in negative $t$ and you
-can go anywhere on the 3D line. For positive $t$, you get only the parts in front of $\mathbf{a}$,
-and this is what is often called a half-line or ray.
+along a ray. Let’s think of a ray as a function $\mathbf{P}(t) = \mathbf{A} + t \mathbf{b}$. (Note:
+throughout these books, we'll use uppercase bold letters for points, and lowercase bold letters for
+vectors.) Here $\mathbf{P}$ is a 3D position along a line in 3D. $\mathbf{A}$ is the ray origin and
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:391]: I'm not sure if this note is necessary. It's a level of formalism that I question for the series

File: books/RayTracingInOneWeekend.html [hunk 2]
```diff
@@ -524,66 +525,66 @@
...
 Ray-Sphere Intersection
 ------------------------
+<div class='together'>
 Recall that the equation for a sphere centered at the origin of radius $R$ is $x^2 + y^2 + z^2 =
 R^2$. Put another way, if a given point $(x,y,z)$ is on the sphere, then $x^2 + y^2 + z^2 = R^2$. If
 the given point $(x,y,z)$ is _inside_ the sphere, then $x^2 + y^2 + z^2 < R^2$, and if a given point
 $(x,y,z)$ is _outside_ the sphere, then $x^2 + y^2 + z^2 > R^2$.
 
-It gets uglier if the sphere center is at $(\mathbf{c}_x, \mathbf{c}_y, \mathbf{c}_z)$:
+It gets uglier if the sphere center is at $(\mathbf{C}_x, \mathbf{C}_y, \mathbf{C}_z)$:
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:540]: I don't know if we discussed this, but I think components of a vector should be stylized as scalars, i.e.
`$(c_x, c_y, c_z)$`

Not sure if they should be upper case or lower case: `$C_x%` or `$c_x$`

File: books/RayTracingInOneWeekend.html [hunk 3]
```diff
@@ -524,66 +525,66 @@
...
 
-  $$ (x-\mathbf{c}_x)^2 + (y-\mathbf{c}_y)^2 + (z-\mathbf{c}_z)^2 = R^2 $$
+  $$ (x-\mathbf{C}_x)^2 + (y-\mathbf{C}_y)^2 + (z-\mathbf{C}_z)^2 = r^2 $$
```
Comment (trevordblack) [inline books/RayTracingInOneWeekend.html:542]: vector component styling

File: books/RayTracingInOneWeekend.html (hunk 1 above)
Comment (hollasch) [reply books/RayTracingInOneWeekend.html:391]: Yeah, I'll pull it.

File: books/RayTracingInOneWeekend.html (hunk 2 above)
Comment (hollasch) [reply books/RayTracingInOneWeekend.html:540]: Agreed — this is a mistake. Will fix.

Comment (hollasch) [reply books/RayTracingInOneWeekend.html:540]: I've kept the scalar components with the same case as their source. Vector components are lowercase, point components are uppercase.

File: books/RayTracingInOneWeekend.html (hunk 3 above)
Comment (hollasch) [reply books/RayTracingInOneWeekend.html]: Done.

**Pull request merged**